src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplLaunchTest.java

```diff
@@ -199,45 +204,15 @@ public void shouldRejectCeManagerFormFromEstabRegionN() {
         "All Northern Ireland calls from CE Managers are to be escalated to the NI management team.");
   }
 
-  private void mockEqLaunchJwe() throws Exception {
-    // Mock out building of launch payload
-    Mockito.when(
-            eqLaunchService.getEqLaunchJwe(
-                eq(Language.ENGLISH),
-                eq(uk.gov.ons.ctp.common.domain.Source.CONTACT_CENTRE_API),
-                eq(uk.gov.ons.ctp.common.domain.Channel.CC),
-                any(),
-                any(),
-                any(),
-                any(),
-                any(),
-                any(),
-                isNull())) // keystore
-        .thenReturn("simulated-encrypted-payload");
-  }
+  private void verifyEqLaunchJwe(boolean individual, String caseType) throws Exception {
+    Mockito.verify(eqLaunchService).getEqLaunchJwe(eqLaunchDataCaptor.capture());
 
-  private void verifyEqLaunchJwe(
-      String questionnaireId, boolean individual, String caseType, FormType formType)
-      throws Exception {
-    Mockito.verify(eqLaunchService)
-        .getEqLaunchJwe(
-            eq(Language.ENGLISH),
-            eq(uk.gov.ons.ctp.common.domain.Source.CONTACT_CENTRE_API),
-            eq(uk.gov.ons.ctp.common.domain.Channel.CC),
-            caseCaptor.capture(),
-            eq(AN_AGENT_ID), // agent
-            eq(questionnaireId),
-            eq(formType.name()),
-            isNull(), // accountServiceUrl
-            isNull(),
-            any()); // keystore
-
-    CaseContainerDTO capturedCase = caseCaptor.getValue();
+    EqLaunchData eqLaunchData = eqLaunchDataCaptor.getValue();
```

Comment:
You have lost some testing verification here.
Please add asserts that each element of the captor value match the expected value (as we have previously, but in the parameter list).